summarizer/summarize.test.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const summarize = require('./summarize');
+
+describe('summarize tests...', () => {
```

Comment:
We need to convert this to Jest format (i.e., `test()` vs. `describe()`)

Reply:
I am unable to next tests in Jest, I use describe to define a scope just in case I wish to add other types of test cases into the file.  It isn't really needed at the moment so I could just remove describe,.